codalab/lib/bundle_cli.py

```diff
@@ -2268,7 +2268,8 @@ def print_host_worksheets(self, info):
     def print_permissions(self, info):
         print('permission: %s' % permission_str(info['permission']), file=self.stdout)
         print('group_permissions:', file=self.stdout)
-        print('  %s' % group_permissions_str(info['group_permissions']), file=self.stdout)
+        if 'group_permissions' in info:
+            print('  %s' % group_permissions_str(info['group_permissions']), file=self.stdout)
```

Comment:
Could we instead just print out an empty value instead of printing nothing? So we could remove the if statement and do this:
```suggestion
            print('  %s' % group_permissions_str(info.get('group_permissions',  [])), file=self.stdout)
```

Reply:
+1 using `info.get()`

Reply:
Hi Ashwin and Jane, thanks for the review. I think the difference is in output, in which,
the previous output will print nothing about `group_permissions` if no group permissions at all;
the current output will be `group_permissions: ` if nothing.
Changes have been made according to the comment.

Reply:
@yshen-google  I see your point. I just ran the changes locally. I am wondering how do you guys think about setting `group_permissions` to `<none>` instead of leaving it blank as follows (which is a little bit less intuitive)?
```
permission: all
group_permissions:
  -
```

Reply:
I am fine with both tbh as long as it avoids the KeywordNotFound error which breaks UX.